packages/airnode-node/src/api/index.ts

```diff
@@ -135,7 +134,7 @@ function verifyRequestId(payload: CallApiPayload): LogsData<ApiCallErrorResponse
 
 export function verifyTemplateId(payload: CallApiPayload): LogsData<ApiCallErrorResponse> | null {
   const { aggregatedApiCall } = payload;
-  if (aggregatedApiCall.type !== 'regular') return null;
+  if (aggregatedApiCall.type === 'http-gateway') return null;
```

Comment:
We should be able to use this method now for Singed data requests, so even if we pass in a templateId to the http request an invalid template would raise an error (if we decide to go that route).